sabnzbd/downloader.py

```diff
@@ -221,6 +223,7 @@ def request_info(self):
             self.request = True
             Thread(target=self._request_info_internal).start()
 
+    @NzbQueueLocker
     def get_article(self):
```

Comment:
```suggestion
    @synchronized(DOWNLOADER_LOCK)
    def get_article(self):
```

Change this one to use the new lock also?

Reply:
Indeed!